src/Symfony/Component/Cache/DoctrineProvider.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache;
+
+use Doctrine\Common\Cache\CacheProvider;
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class DoctrineProvider extends CacheProvider
```

Comment:
no it does not, because the opcache adapter is read-only and so caching annotations won't work properly. You cannot ensure that the annotation reader will never miss the cache after the cache warmup, unless you load annotations from **all** doc-comment of the code base during the cache warmup (which means all classes, all methods, all functions and all inline doc-comment), which does not make sense (as 99% of the cache will never be needed, which would make memory usage worse here, but you cannot know whether a particular reflector is in the 1% or the 99%)